playbooks/nfs.yml

```diff
@@ -26,6 +26,15 @@
     nftables_path: /etc/nftables
     nftables_conf: /etc/sysconfig/nftables.conf
   tasks:
+    - name: Set custom cifmw PATH reusable fact
+      tags:
+        - always
+      when:
+        - cifmw_path is not defined
+      ansible.builtin.set_fact:
+        cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
```

Comment:
so this is going to be used somewhere that is not running the bootstrap play https://github.com/openstack-k8s-operators/ci-framework/blob/89bdefa600aa5c6b4c86fc070d4656899efed094/playbooks/01-bootstrap.yml#L11 
?

Reply:
This playbook/hook runs on compute hosts[1] and not on ansible controller and this var is not available for it.

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/playbooks/nfs.yml#L19